minari/wrappers/error.py

```diff
@@ -0,0 +1,10 @@
+"""Set of Error classes for gymnasium."""
+
+
+class Error(Exception):
```

Comment:
Do we need this? I know it exists in gymnasium which is already an interesting design choice (inherited from gym), but in that case it's probably better to just import it from there. Or just subclass `Exception` directly.